Gulpfile.mjs

```diff
@@ -292,9 +292,12 @@ function buildRollup(packages, targetBrowsers) {
         plugins: [
           rollupBabelSource(),
           rollupReplace({
-            "process.env.NODE_ENV": JSON.stringify(process.env.NODE_ENV),
-            BABEL_VERSION: JSON.stringify(babelVersion),
-            VERSION: JSON.stringify(version),
+            preventAssignment: true,
+            values: {
```

Comment:
This option is not documented but it's nice since it separates options from replacements.
I opened a PR to document it (https://github.com/rollup/plugins/pull/814)